draft-ietf-httpbis-http2bis.xml

```diff
@@ -769,7 +545,8 @@ HTTP Frame {
                   </g>
                 </g>
               </svg>
-            </artwork>
+<![CDATA[
+            ]]></artwork>
```

Comment:
These `CDATA` insertions (here and above) are a bit of a surprise. Why are they here?

Reply:
Oh, that's a bug in the script.  Well, not so much a bug as a limitation that comes as a result of `<artwork>` sometimes containing ASCII art and other times containing `<svg>`.  Thanks for catching that, I'll fix it.